src/contracts/pods/EigenPodManager.sol

```diff
@@ -207,6 +422,124 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
     }
 
     // INTERNAL FUNCTIONS
+    function _queueWithdrawal(
+        address podOwner,
+        uint256 amountWei,
+        address withdrawer,
+        bool undelegateIfPossible
+    ) 
+        internal
+        returns (bytes32)
+    {
+        require(amountWei > 0, "EigenPodManager._queueWithdrawal: amount must be greater than zero");
+
```

Comment:
TODO: make sure there's no edge case where we hit this and cause an unexpected revert

Reply:
I'm OK merging this PR without addressing this particular issue. I will give myself a TODO elsewhere.

Reply:
This check (or at least a similar one) was actually the source of a bug, and has since been removed